egs/wsj/s5/steps/libs/nnet3/train/chain_objf/acoustic_model.py

```diff
@@ -426,13 +426,16 @@ def compute_preconditioning_matrix(dir, egs_dir, num_lda_jobs, run_opts,
     common_lib.force_symlink("../lda.mat", "{0}/configs/lda.mat".format(dir))
 
 
-def prepare_initial_acoustic_model(dir, run_opts, srand=-1):
-    """ Adds the first layer; this will also add in the lda.mat and
-        presoftmax_prior_scale.vec. It will also prepare the acoustic model
-        with the transition model."""
-
-    common_train_lib.prepare_initial_network(dir, run_opts,
-                                             srand=srand)
+def prepare_initial_acoustic_model(dir, run_opts, srand=-1, input_mdl=None):
+    """ Adds the first layer; It will also prepare the acoustic model
+        with the transition model.
+        If input_mdl is specified, no initial network preparation(adding
+        first layer) is done on that and this model is prepared instead of
+        '0.raw' acoustice model with the transition model.
```

Comment:
spelling.  maybe you should run a spell-checker because there are too many of these problems.